parsl/configs/Azure.py

```diff
@@ -26,7 +26,6 @@
     executors=[
         HighThroughputExecutor(
             label='azure_single_node',
-            address=address_by_query(),
```

Comment:
Should remove the import `address_by_query` above.

Reply:
ZZ, Azure uses address_by_query further down the config for the `storage_access` option for `RSyncStaging`. So we shouldn't remove the import.

Reply:
Oh, I missed that.